api/clients/Admin/src/main.js

```diff
@@ -230,7 +230,7 @@ function refreshView(){
 
                 if (sensor.installed_at) {
                     measurementsPs.push(new Promise(function (resolve) {
-                        api.getPlaceMeasurements({id: sensor.installed_at, types: ['wifi']})
+                        api.measurementsPlaces({ids: [sensor.installed_at], types: ['wifi']})
```

Comment:
Maybe i'm being too much into details here, but it looks like we have endpoints with `get` as a prefix, and others without, even if they do **get** something too...

It's just a piece of thought.